src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCSVFileRecord.java

```diff
@@ -75,7 +75,7 @@
      * Constructs a simple reader to parse data from a delimited file with the given encoding.
      * 
      * @param fileToParse
-     *        File to parse data from
+     *        File to parse data from, please ensure special regex characters are properly escaped.
```

Comment:
Does this mean the file name can contain regex characters? I'm confused by this change.

Reply:
This should be under the delimiter param, will change